crates/ruff_linter/src/rule_selector.rs

```diff
@@ -213,6 +213,8 @@ impl RuleSelector {
             || (preview_enabled && (matches!(self, RuleSelector::Rule { .. }) || !preview_require_explicit))
             // Deprecated rules are excluded in preview mode unless explicitly selected
             || (rule.is_deprecated() && (!preview_enabled || matches!(self, RuleSelector::Rule { .. })))
+            // Removed rules are included if explicitly selected but will error downstream
+            || (rule.is_removed() && matches!(self, RuleSelector::Rule { .. }))
```

Comment:
Do these show up in the JSON Schema? (I think they shouldn't.)

Reply:
Yeah let's remove them from there..